Server/app.js

```diff
@@ -0,0 +1,29 @@
+const express = require('express')
+const app = express()
+const cors = require('cors');
+
+const webpages = require("./data")
+
+app.use(cors());
+
+app.get('/', (req, res) => res.send('Hello World!!!!'))
+
+app.get('/searchresults', (req, res) => res.send(webpages) )
```

Comment:
Use a query in the request to the API to get  specific webpage back. 
A query looks like app.get('/'searchresults?search=google).
In app.js, req.query will give you an object with the query in it, with this example the object would be {search: 'google'}. You can then use this to find the webpage that someone wants.